content/development-guide/api/command.md

```diff
@@ -95,13 +89,13 @@ public IChildCommand [] ChildCommands { get; } = new IChildCommand []
 Now `/sample joke` should work.
 
 ### Run method in Plugin
-Just add the following to your SampleCommand
+Just add the following to your `SampleCommand`.
 
 ```csharp
 private MyPlugin plugin;
 
-public SampleCommand (IPlugin plugin)
+public SampleCommand (IPlugin _plugin)
```

Comment:
Underscore method parameters are not permitted in our codestyle

Reply:
Yeah, the same mistake I did in other pull request xD